core/sodasql/scan/dialect.py

```diff
@@ -133,7 +134,7 @@ def create_connection(self):
     def sql_columns_metadata_query(self, table_name: str) -> str:
         raise RuntimeError('TODO override and implement this abstract method')
 
-    def sql_tables_metadata_query(self, limit: str = 10, filter: str = None):
+    def sql_tables_metadata_query(self, limit: int = None, filter: str = None):
```

Comment:
Shouldnt we mark these params with `Optional[X]` in case of passing `None` as default? 🤔 This is repeating in many places

Reply:
Yup, I'll update the type hint